evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -40,8 +44,16 @@ def validate(self, X, y):
             >>> X = pd.DataFrame({})
             >>> y = pd.Series([0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1])
             >>> target_check = ClassImbalanceDataCheck(threshold=0.10)
-        >>> assert target_check.validate(X, y) == {"errors": [{"message": "The number of instances of these targets is less than 2 * the number of cross folds = 6 instances: [0]", "data_check_name": "ClassImbalanceDataCheck", "level": "error"}],\
-                                                       "warnings": [{"message": "The following labels fall below 10% of the target: [0]", "data_check_name": "ClassImbalanceDataCheck", "level": "warning"}]}
+        >>> assert target_check.validate(X, y) == {"errors": [{"message": "The number of instances of these targets is less than 2 * the number of cross folds = 6 instances: [0]",\
```

Comment:
This is a great format! My only comment is that Enums are not actually json-serializable and the issue specifically mentions a "JSON-friendly" message format. I think doing `.name` on the code will be really useful for users! 

![image](https://user-images.githubusercontent.com/41651716/99995819-f12d1d80-2d88-11eb-8c43-156eedf47bd2.png)